init/clean.js

```diff
@@ -1,31 +1,27 @@
-const { readFile, writeFile } = require('fs')
-const { promisify } = require('util')
+import { readFile, writeFile } from 'fs/promises'
```

Comment:
this is not available in node 12 if we have to support it we need to write it differently

```suggestion
import { promises as fs } from 'fs'

const { readFile, writeFile } = fs
```

Reply:
Good catch! Fixed at f6161b0732ecead12e1adcb7c12184708fde2ed8

Reply:
Should we add a matrix pipeline with nod 12 in there to catch such issues?

Reply:
Yes!
Please note we do not have any automated tests on the `init` logic at the moment, only on the `src/index.js` (and it is just a smoke test).